lib/mix/lib/mix/generator.ex

```diff
@@ -97,7 +97,7 @@ defmodule Mix.Generator do
 
   ## Examples
 
-      iex> Mix.Generator.copy_template("source/gitignore", ".gitignore", [project_path: path])
```

Comment:
:project_path is not listed among the valid options, and does not seem to be used

Reply:
I made a mistake here. I think project_path was meant to be an assign. I'll make a quick followup PR to put this back in.

Reply:
Ah, good call. Maybe we should do this:

    iex> assigns = [project_path: ...]

for clarity?